supabase/migrations/20230924193504_label_changes.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE IF NOT EXISTS label_changes (
+    id serial PRIMARY KEY,
+    created_at timestamptz NOT NULL,
+    updated_at timestamptz NOT NULL,
+    username text NOT NULL,
+    repository text NOT NULL,
+    label_from text NOT NULL,
+    label_to text NOT NULL,
+    approved boolean NOT NULL
```

Comment:
I think we will also need a command to approve.

Reply:
Yea, i am working on that now